roles/edpm_nova_compute/meta/argument_specs.yml

```diff
@@ -3,4 +3,924 @@ argument_specs:
   # ./roles/edpm_nova_compute/tasks/main.yml entry point
   main:
     short_description: The main entry point for the edpm_nova_compute role.
-    options: {}
+    options:
+      service_net_map:
```

Comment:
edpm_service_net_map is the paramter used by the role and service_net_map is the default that it is initalised form